packages/core/src/App/Containers/Layout/header.jsx

```diff
@@ -98,6 +102,7 @@ class Header extends React.Component {
                     </div>
                 </div>
                 <RealAccountSignup />
+                <SetAccountCurrencyModal />
```

Comment:
Please place this in `AppModals`.

Reply:
I did initially, u can refer to my commit history, but the onExit `CSSTransitionGroup` transitions doesn't work when I had it there.